src/main/java/HomePageAmazon.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeMethod;
+
+
+public class HomePageAmazon {
+
+    WebDriver driver;
+
+    public HomePageAmazon(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void putTextOncjeckBox (){
```

Comment:
can you rename the function to something more clear?
Please use an argument on the function, for example
`search("Nintendo")`